examples/KubernetesProvider/quickstart-webapp-feature-flag/MyWebApp/Pages/Beta.cshtml.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace MyWebApp.Pages
+{
+    [FeatureGate("Beta")]
```

Comment:
Is this only a simple enable/disable scenario? Could we add more scenarios for covering client filters?